vcpkg/get_started/setup-vcpkg.md

```diff
@@ -1,86 +1,62 @@
-zone_pivot_group_filename: zone-pivot-groups.json
-zone_pivot_groups: operating-systems
-author: JavierMatosD
-ms.topic: tutorial
-ms.date: 08/31/2023
-ms.author: javiermat
-ms.prod: vcpkg
-ROBOTS: NOINDEX
-#CustomerIntent: As a user, I want to install vcpkg for my operating system.
----
+## 1 - Install vcpkg
 
-# Tutorial: Install vcpkg
+1. Clone the repository
 
-This tutorial provides a step-by-step guide for installing vcpkg on a Windows, macOS, or Linux system. Follow this tutorial if you want to set up vcpkg on a development machine or continuous integration system. If you are a Visual Studio or Visual Studio Code user, see the following tutorials instructions to use vcpkg within the editor:
+    The first step is to clone the vcpkg repository from GitHub. The repository contains scripts to acquire the vcpkg executable and a registry of curated open-source libraries maintained by the vcpkg community. To do this, run:
 
-- [Use vckg from Visual Studio IDE]()
-- [Use vckg from Visual Studio Code]()
+    ```console
+    git clone https://github.com/microsoft/vcpkg.git
+    ```
 
-## Prerequisites
+    The vcpkg open-source registry is a set of over 2,000 free libraries that have been tested together and optimized to work smoothly with vcpkg. While the vcpkg repo does not contain the source code for these libraries, it does contain build recipes and metadata about each library. These files will be included when vcpkg is installed to your system.
 
-- A terminal to work in.
-- `Git`
+2. Run the bootstrap script
 
-## 1 - Clone the repository
+    Now that you have cloned the vcpkg repository, navigate to the `vcpkg` directory and execute the bootstrap script:
 
-The first step is to clone the vcpkg repository from GitHub. The repository contains scripts to acquire the vcpkg executable and a registry of curated open-source libraries maintained by the vcpkg community. To do this, run:
+    ::: zone pivot="os-windows"
 
-```console
-git clone https://github.com/microsoft/vcpkg.git
-```
+    ```console
+    cd vcpkg && ./bootstrap-vcpkg.bat
+    ```
 
-The vcpkg open-source registry is a set of over 2,000 free libraries that have been tested together and optimized to work smoothly with vcpkg. While the vcpkg repo does not contain the source code for these libraries, it does contain build recipes and metadata about each library. These files will be included when vcpkg is installed to your system.
+    ::: zone-end
+    ::: zone pivot="os-linux,os-macos"
 
-## 2 - Run the bootstrap script
+    ```console
+    cd vcpkg && ./bootstrap-vcpkg.sh
+    ```
 
-Now that you have cloned the vcpkg repository, navigate to the `vcpkg` directory and execute the bootstrap script:
+    ::: zone-end
 
-::: zone pivot="os-windows"
+    The bootstrap script performs prerequisite checks, downloads the vcpkg executable, and manages telemetry settings.
 
-```console
-cd vcpkg && ./bootstrap-vcpkg.bat
-```
+3. Enable terminal auto-completion (optional)
```

Comment:
I think this is too much for a getting started guide. Can this go somewhere else as an intermediate "getting more comfortable with vcpkg"?